lib/scarpe/stack.rb

```diff
@@ -15,7 +17,7 @@ def element
     private
 
     def style
-      styles = {}
+      styles = super
```

Comment:
Am I right that the super here allows us to grab the `style` from the mixin module?

Reply:
Exactly! What might happen later on down the line is that another mixing that changes styles gets added too, then the behaviour can become tricky